build-system/externs/amp.extern.js

```diff
@@ -242,6 +242,7 @@ window.__AMP_PARENT;
 window.__AMP_WEAKREF_ID;
 window.__AMP_URL_CACHE;
 window.__AMP_LOG;
+window.__AMP_OT_INSERTED;
```

Comment:
@samouri can you confirm I am doing this right? I want to make sure it doesn't get mangled across multiple instances of the same extension, nor multiple extensions.

Reply:
never mind :) decided to queryselect instead